nf_core/modules/bump_versions.py

```diff
@@ -149,9 +150,12 @@ def bump_module_version(self, module: NFCoreModule):
                     return False
 
                 patterns = [
-                    (bioconda_packages[0], f"bioconda::{bioconda_tool_name}={last_ver}"),
-                    (r"quay.io/biocontainers/{}:.*".format(bioconda_tool_name), docker_img),
-                    (r"https://depot.galaxyproject.org/singularity/{}:.*".format(bioconda_tool_name), singularity_img),
+                    (bioconda_packages[0], f"'bioconda::{bioconda_tool_name}={last_ver}'"),
+                    (r"quay.io/biocontainers/{}:[^\'\"\s]*".format(bioconda_tool_name), docker_img),
```

Comment:
```suggestion
                    (r"quay.io/biocontainers/{}:[^'\"\s]+".format(bioconda_tool_name), docker_img),
```

I don't think that you need to escape the `'` as well...(?) Also probably want 1 or more characters, not 0 or more. Otherwise looks good!

Reply:
Good point, thanks!